### ISSUE TITLE
Active tab and Hovered tab settings not saving

### ISSUE THREAD

**Issue by TeemuKioski** [MurzNN/TST-Colored-tabs#10]
**System:** FF 70.0, MacOS 10.14.6, plugin versio 0.6. 
FF default theme, and tested with other plugins disabled (apart from TST, obviously). 

The "Active tab" and "Hovered tab" settings don't get applied when you change their settings from the empty defaults and click "Save and apply". If you leave or close the plugin settings page and come back the settings have defaulted to empty. Apparently something is causing the settings to not save at all.

**Comment by moglee** [comment]
I think this is happening because settings are saved in a nested format but loaded via a underscore separator. I believe this can be fixed by modifying background.js line 5-14 to contain:

```javascript
const DEFAULT_SETTINGS = {  
  saturation: 60,  
  lightness: 70,  
  colors: 15,  
  active: {  
      saturation: 60,  
      lightness: 60,  
      bold: true,  
    },  
  hover: {  
      saturation: 60,  
      lightness: 65  
    }  
};  
```
Love the extension by the way :)

**Comment by artemklevtsov** [comment]
Fix already in PR (#7).

**Comment by MurzNN** [comment]
Thans for PR! ...and sorry for too long delay. Extension was be on refactoring for adding custom host colors, and now I publish new version where this issue must be also fixed.
